### DISCUSSION TITLE
"Review pull requests" walk-through re: Step 3 and 4 questions.

### DISCUSSION THREAD

**Discussion by BecAndCall** [skills/.github#9] [Q&A]
Hello, Skills team.

Everything went pretty smoothly, overall, but I did not see the "Insert a suggestion button" in Step 3. This may be why I did not see the "Commit suggestion" option in Step 4.

Can anyone screen shot or advise what I missed? I'd appreciate it. I'll walk through again until I get this right. 👍 

Cheers, Bec

**Comment by MrKrishnaAgarwal** [comment]
Please see the instructions of the course.
and if the issue persists, mention me here.

**Reply by BecAndCall** [reply]
@MrKrishnaAgarwal  Thank you for the quick reply.  I did see the instructions but I DIDN'T see the ""Insert a suggestion button" in Step 3 anywhere.  Maybe my screen is too small or the UI has changed. Can you please send me a screen shot showing the "Insert a suggestion button"?

**Reply by sinsukehlab** [reply]
I wasn't able to find the **Insert a suggestion** button, either.
I found an **Add a suggestion** button instead.
I have opened an issue in skills/review-pull-requests.
See: [my issue](https://github.com/skills/review-pull-requests/issues/7)

**Reply by Antonio072** [reply]
Hi, I think that they mean, it would be better to add this icon also, because it is a litle bit confusing when trying to find those exact words.
![imagen](https://user-images.githubusercontent.com/31423917/176461944-05402e2b-531b-4563-9758-d6ae03950940.png)

**Reply by sinsukehlab** [reply]
I have added some commits in [my PR](https://github.com/skills/review-pull-requests/pull/8). Thanks.

**Reply by sinsukehlab** [reply]
My PR was finally merged just now.

**Comment by MrKrishnaAgarwal** [comment]
Course Name Please?

**Comment by sinsukehlab** [comment]
[This PR](https://github.com/skills/review-pull-requests/pull/8) answers the question.
